relay-server/src/actors/health_check.rs

```diff
@@ -96,9 +96,15 @@ impl HealthCheckService {
                     return false;
                 }
 
-                compat::send(Aggregator::from_registry(), AcceptsMetrics)
-                    .await
-                    .unwrap_or(false)
+                // NOTE: usually this unwrap is hidden in the `from_registry` function on the
+                // specific service or actor. But since the `AggregatorService` is in the different
+                // crate we cannot easily implement that pattarn and for now we have to use
+                // REGISTRY dirrectly and get the aggregator service out of it.
```

Comment:
```suggestion
                // specific service or actor. But since the `AggregatorService` is in the different
                // crate we cannot easily implement that pattern and for now we have to use
                // REGISTRY directly and get the aggregator service out of it.
```